scripts/package

```diff
@@ -15,5 +15,11 @@ fi
 cp ../bin/longhorn* .
 cp ../bin/longhorn jivactl
 #cp /usr/src/tgt/pkg/tgt_*.deb .
-docker build -t ${REPO}/jiva:${TAG} .
+if [ ${ARCH} == "linux_arm64" ]
+then
+  DOCKERFILE=Dockerfile_build_arm64
+else
+  DOCKERFILE=Dockerfile_build_amd64
+fi
+docker build -f ${DOCKERFILE} -t ${REPO}/jiva:${TAG} .
```

Comment:
Can we add suffix arm64 in the TAG only for the arm64 builds ?

Reply:
cc: @kmova

Reply:
Good point @utkarshmani1997 -- @Wangzihao18 -- can we change the above to generate the following docker images based on the ARCH. 

```
For ARM64: 
  DOCKERFILE=Dockerfile_build_arm64
  docker build -f ${DOCKERFILE} -t ${REPO}/jiva-${ARCH}:${TAG} .

For AMD
  DOCKERFILE=Dockerfile_build_arm64
  docker build -f ${DOCKERFILE} -t ${REPO}/jiva:${TAG} .
  #Above is to maintain backward compatibility with current versions. 
```

Reply:
ok.  @kmova , @utkarshmani1997. I am on vacation.  I will go on the work four days later : ) Thanks for your suggestions.

Reply:
Hi, @kmova , @utkarshmani1997 . I have change the image name in the scripts/package and scripts/package_debug for different arch . Pls give a review again : )